lhotse/dataset/unsupervised.py

```diff
@@ -10,7 +10,8 @@
 class UnsupervisedDataset(Dataset):
     """
     Dataset that contains no supervision - it only provides the features extracted from recordings.
-    The returned features are a torch Tensor of shape (T x F).
+    The returned features are a :class:`torch.Tensor` of shape ``(T x F)``, where T is the number of frames,
+    and F is the feature dimension.
```

Comment:
How will you implement collate_fn?

Reply:
I'm considering a scenario where a custom collate_fn is not necessary; we'd instead truncate/pad all the samples into a number of duration bins, and implement a [Sampler](https://pytorch.org/docs/stable/data.html?highlight=sampler#torch.utils.data.Sampler) that constructs batches only from same duration samples. Then we'd only need to implement collate_fn if we opt into returning custom type objects from dataset (like graphs); even then, the only modification to the default collate_fn would be "informing it" how to batch these objects together.

Reply:
"truncate/pad all the samples into a number of duration bins" -> I meant in the CutSet